From 4f8ae740cad41aa2fa7b593c7222fdb8d4fa57cd Mon Sep 17 00:00:00 2001
From: Leonid Komarianskyi <leonid_komarianskyi@epam.com>
Date: Fri, 21 Jan 2022 10:49:50 +0200
Subject: [PATCH 3/8] [systemd] Allow getaatr access to cgroup domain

Signed-off-by: Leonid Komarianskyi <leonid_komarianskyi@epam.com>
---
 policy/modules/system/systemd.te | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/policy/modules/system/systemd.te b/policy/modules/system/systemd.te
index 7b2d359b7..9bc636864 100644
--- a/policy/modules/system/systemd.te
+++ b/policy/modules/system/systemd.te
@@ -1281,6 +1281,9 @@ systemd_log_parse_environment(systemd_sessions_t)
 mls_file_read_to_clearance(systemd_sessions_t)
 mls_file_write_all_levels(systemd_sessions_t)
 
+fs_getattr_cgroup(systemd_sessions_t)
+fs_search_cgroup_dirs(systemd_sessions_t)
+
 #########################################
 #
 # Sysusers local policy
@@ -1422,6 +1425,8 @@ mls_file_read_all_levels(systemd_tmpfiles_t)
 mls_file_downgrade(systemd_tmpfiles_t)
 mls_file_upgrade(systemd_tmpfiles_t)
 
+fs_getattr_cgroup(systemd_tmpfiles_t)
+
 userdom_manage_user_runtime_root_dirs(systemd_tmpfiles_t)
 userdom_relabel_user_runtime_root_dirs(systemd_tmpfiles_t)
 
@@ -1577,6 +1582,8 @@ fs_getattr_tmpfs(systemd_user_runtime_dir_t)
 fs_list_tmpfs(systemd_user_runtime_dir_t)
 fs_unmount_tmpfs(systemd_user_runtime_dir_t)
 fs_relabelfrom_tmpfs_dirs(systemd_user_runtime_dir_t)
+fs_getattr_cgroup(systemd_user_runtime_dir_t)
+fs_search_cgroup_dirs(systemd_user_runtime_dir_t)
 
 kernel_read_kernel_sysctls(systemd_user_runtime_dir_t)
 
-- 
2.25.1

