From 2bbb23cec5355c7289db6d91e445d10ba41c641e Mon Sep 17 00:00:00 2001
From: Valerii Chubar <valerii_chubar@epam.com>
Date: Fri, 14 Aug 2020 14:31:27 +0300
Subject: [PATCH] Respect go static flags

---
 Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 2b9c4c08..b3d9022b 100644
--- a/src/import/Makefile
+++ b/src/import/Makefile
@@ -25,8 +25,9 @@ ifeq ($(shell $(GO) env GOOS),linux)
 endif
 GO_BUILD := $(GO) build $(MOD_VENDOR) $(GO_BUILDMODE) $(EXTRA_FLAGS) -tags "$(BUILDTAGS)" \
 	-ldflags "-X main.gitCommit=$(COMMIT) -X main.version=$(VERSION) $(EXTRA_LDFLAGS)"
-GO_BUILD_STATIC := CGO_ENABLED=1 $(GO) build $(MOD_VENDOR) $(EXTRA_FLAGS) -tags "$(BUILDTAGS) netgo osusergo" \
-	-ldflags "-w -extldflags -static -X main.gitCommit=$(COMMIT) -X main.version=$(VERSION) $(EXTRA_LDFLAGS)"
+GO_BUILD_STATIC :=CGO_ENABLED=1 $(GO) build $(EXTRA_FLAGS) \
+	-tags "$(BUILDTAGS) netgo osusergo static_build" -installsuffix netgo \
+	-ldflags "-w -extldflags -static -X main.gitCommit=${COMMIT} -X main.version=${VERSION} $(EXTRA_LDFLAGS)"
 
 .DEFAULT: runc
 
-- 
2.17.1

